internal/state/graph/backend_refs.go

```diff
@@ -169,16 +169,16 @@ func validateBackendRef(
 ) (valid bool, cond conditions.Condition) {
 	// Because all errors cause same condition but different reasons, we return as soon as we find an error
 
-	if ref.Group != nil && !(*ref.Group == "core" || *ref.Group == "") {
-		valErr := field.NotSupported(path.Child("group"), *ref.Group, []string{"core", ""})
-		return false, conditions.NewRouteBackendRefUnsupportedValue(valErr.Error())
-	}
-
 	if ref.Kind != nil && *ref.Kind != "Service" {
```

Comment:
because Kind depends on the group (for example, the fact that the kind is Service (valid) is meaningless if the group is not core), I think the current logic is correct. Perhaps we can just report `conditions.NewRouteBackendRefInvalidKind` for the group error as well, without changing the order, since the group is kind of related to the kind anyway and there is nothing in the spec written about the group validation.

@kate-osborn @sjberman  wonder what you think

Reply:
> Perhaps we can just report conditions.NewRouteBackendRefInvalidKind for the group error as well

That's actually what I had first but it didn't feel correct as it looks like the test is checking specifically for Kind not Group - even though the Group is also incorrect - but your explanation makes it make more sense! I'll change it.